server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -978,8 +979,20 @@ synchronized void completeClose(boolean saveState, boolean completeClose) throws
       activeScan.interrupt();
     }
 
+    long lastLogTime = System.nanoTime();
+
     // wait for reads and writes to complete
     while (writesInProgress > 0 || !activeScans.isEmpty()) {
+
+      if (log.isDebugEnabled()
+          && System.nanoTime() - lastLogTime > Duration.ofSeconds(60).toNanos()) {
```

Comment:
Duration.ofSeconds() is overkill, as it would create wrapper objects just to do a bit of math. It's much better to just use `SECONDS.toNanos(60)` (static import TimeUnit.SECONDS).

Reply:
I look into that and see what it looks like.  I wrote that code intentionally like that thinking it was more readable and that in this particular case performance did not matter because its only called when debug log is enabled and its in a loop that sleeps (a bit later in the loop it sleeps 50ms).

Reply:
Updated in 65193af to use TimeUnit and I think that looks good.  I did not do the static import for two reasons.   First the if stmt was shortened using TimeUnit and fit on a single line w/o doing the static import.  Second its the only use of TimeUnit.SECONDS in the class, so did not want to add a static import for a single use.